### DISCUSSION TITLE
v1.8 Broken for me

### DISCUSSION THREAD

**Discussion by nebadon2025** [Heroic-Games-Launcher/HeroicGamesLauncher#508] [General]
I upgraded to v1.8 and its zoomed to like 200% and it will not let me resize the window at all, also it keeps telling me there is a new version, yet it points me to the same version I already have, i can't even access the whole interface so its basically 100% unuseable now I will include system specs below as well.  I am running in 3840x2160 resolution with 200% desktop scaling in KDE

![heroic1_8_borked_01](https://user-images.githubusercontent.com/693958/124321750-3948c580-db4c-11eb-90da-5eb8eaeea9d7.png)
![heroic1_8_borked_02](https://user-images.githubusercontent.com/693958/124321752-3b128900-db4c-11eb-9325-9a92e0165d37.png)
![heroic1_8_borked_03](https://user-images.githubusercontent.com/693958/124321757-3c43b600-db4c-11eb-9e73-1ff5b3d8aa5a.png)

[System]
OS:              Ubuntu 20.10 groovy
Arch:            x86_64
Kernel:          5.12.14-xanmod1
Desktop:         KDE
Display Server:  x11

[CPU]
Vendor:          AuthenticAMD
Model:           AMD Ryzen 9 3900X 12-Core Processor
Physical cores:  12
Logical cores:   24

[Memory]
RAM:             62.8 GB
Swap:            32.0 GB

[Graphics]
Vendor:          AMD
OpenGL Renderer: AMD Radeon RX 5700 XT (NAVI10, DRM 3.40.0, 5.12.14-xanmod1, LLVM 12.0.0)
OpenGL Version:  4.6 (Compatibility Profile) Mesa 21.2.0-devel (git-25ad699 2021-07-02 groovy-oibaf-ppa)
OpenGL Core:     4.6 (Core Profile) Mesa 21.2.0-devel (git-25ad699 2021-07-02 groovy-oibaf-ppa)
OpenGL ES:       OpenGL ES 3.2 Mesa 21.2.0-devel (git-25ad699 2021-07-02 groovy-oibaf-ppa)
Vulkan:          Supported

**Comment by flavioislima** [comment]
That's weird. The issue about showing that there is a new version is known and we will release a fix soon.
Can you start heroic from the terminal using
`heroic --disable-gpu`?
What's the last version that was working? 1.7.2?

**Comment by nebadon2025** [comment]
Same results with --disable-gpu from the console, and yes I was previously running 1.7.2 I always update immediately when prompted.
